### PR TITLE
Feature/fks 107

### PR DESCRIPTION
**Pull request by Evil00787**
NOTHING has changed in caregiver/plans_page - I literally copied this file and it shows changes :(
Sorry if this code is thrash <I'm out>

### CODE REVIEW

File: lib/widgets/app_header.dart [hunk 1]
```diff
@@ -24,32 +24,47 @@ class HeaderActionButton {
...
+		text: text,
+		headerActionButtons: headerActionButtons,
+		headerType: AppHeaderType.normal
+	);
+	AppHeader.widget({String title, String text, List<HeaderActionButton> headerActionButtons, Widget appHeaderWidget}) : this(
+		title: title,
+		text: text,
+		headerActionButtons: headerActionButtons,
+		headerType: AppHeaderType.widget,
+		appHeaderWidget: appHeaderWidget
```
Comment (stasgora) [inline lib/widgets/app_header.dart:56]: Where is that type used?

File: lib/pages/child/plan_in_progress_page.dart [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+						pinnedTitle: '$_pageKey.header.title',
+						flexibleSpaceWidget: ItemCard(
+							title: "Sprzątanie pokoju w odstępach kilkudziesięciukilku minutowych oraz wielogodzinowych melancholijnych spotkaniach przy otwartym oknie pośród gwiazd Betlejem skąpanych w mroku piekła i nieba oraz genialnym muzykowaniem z najlepszymi ludźmi",
+							subtitle: "Co każdy poniedziałek, wtorek i czwartek",
+							isActive: true,
+							progressPercentage: 0.4,
+							chips:
+							<Widget>[AttributeChip.withIcon(
+								icon: Icons.description,
+								color: Colors.lightGreen,
```
Comment (stasgora) [inline lib/pages/child/plan_in_progress_page.dart:34]: Use AppColors

**Comment by Evil00787** [comment]
Moved back to static header - please check again

File: lib/widgets/app_header.dart [hunk 3]
```diff
@@ -229,6 +247,50 @@ class AppHeader extends StatelessWidget {
...
+							  children: <Widget>[
+							  	// TODO: change this widget to widget made by Miko
+							    Padding(
+							      padding: const EdgeInsets.symmetric(horizontal: 8.0),
+							      child: IconButton(icon: Icon(Icons.help, color: Colors.white, size: 26,), onPressed: null,),
+							    ),
+							  ],
+							)
+						]
+					),
```
Comment (stasgora) [inline lib/widgets/app_header.dart:281]: Those Columns & Row are redundant as thay have only one child / children could be merged into the Row higher up
```suggestion
							BackIconButton(),
							Text(
								AppLocales.of(context).translate(title),
								style: Theme.of(context).textTheme.headline1.copyWith(color: Colors.white)
							),
							Padding(
					      padding: const EdgeInsets.symmetric(horizontal: 8.0),
					      child: IconButton(icon: Icon(Icons.help, color: Colors.white, size: 26,), onPressed: null,),
					    ),
						]
					),
```

File: lib/pages/child/plan_in_progress_page.dart [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+			children: [AppHeader.widget(title: '$_pageKey.header.title', appHeaderWidget: ItemCard(
+				title: "Sprzątanie pokoju",
+				subtitle: "Co każdy poniedziałek, środę, czwartek i piątek",
+				isActive: true,
+				progressPercentage: 0.4,
+				activeProgressBarColor: AppColors.childActionColor,
+				chips:
+				<Widget>[AttributeChip.withIcon(
+					icon: Icons.description,
+					color: Colors.lightGreen,
```
Comment (stasgora) [inline lib/pages/child/plan_in_progress_page.dart:33]: AppColors

File: lib/pages/child/plan_in_progress_page.dart [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+					content: AppLocales.of(context).translate('page.childSection.panel.content.taskProgress', {'NUM_TASKS': 2, 'NUM_ALL_TASKS': 5})
+				)],
+			),),
```
Comment (stasgora) [inline lib/pages/child/plan_in_progress_page.dart:36]: Formatting

File: lib/widgets/app_header.dart [hunk 6]
```diff
@@ -229,6 +247,50 @@ class AppHeader extends StatelessWidget {
...
+						mainAxisAlignment: MainAxisAlignment.spaceBetween,
+						crossAxisAlignment: CrossAxisAlignment.center,
+						children: <Widget>[
+							Column(
+							  children: <Widget>[
+							    Row(
+							      children: <Widget>[
+							        BackIconButton(),
+											Text(
+												AppLocales.of(context).translate(title),
```
[the remark this answers is not in the document]
Comment (stasgora) [reply lib/widgets/app_header.dart:264]: Handle that while you add the menu

**Pull request merged**